package.json

```diff
@@ -11,6 +11,12 @@
     "type": "git",
     "url": "git+https://github.com/Microsoft/pxt-microbit.git"
   },
+  "scripts": {
+    "start": "npm run build && npm run serve",
```

Comment:
might be worth discussing the trade-offs here? i.e. that we're not using pxt serve, so we don't get hot-reloading (or the API), but we also avoid issues with differing between dev and prod.

Reply:
Nit: think you might have missed this one. Not important.

Reply:
> This task needs to be run each time you make a change. Which is slow, but is necessary so that we emulate prod.

Not enough?

Reply:
Ah, yeah, good, but I would have also mentioned that the 'normal' way of doing this is to use 'pxt serve', but we've found it frequently inconsistent with what staticpkg produces. i.e. if someone wants to get adventurous in the future, warn them about the landmine.